tin-drink/src/App.test.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
+function stupidFunction(x) {
+  var y = x+5;
```

Comment:
Unexpected var, use let or const instead.
'y' is assigned a value but never used.